ragaai_catalyst/tracers/utils/rag_trace_json_converter.py

```
@@ -0,0 +1,197 @@
+                try:
+                    additional_metadata["tokens"]["prompt"] = span["attributes"]["llm.token_count.prompt"]
+
+                except:
+                    logger.warning("Warning: prompt token not found. using tiktoken to get tokens.")
+                    additional_metadata["tokens"]["prompt"] = num_tokens_from_messages(
+                        model=additional_metadata["model_name"],
+                        message=prompt
+                    )
+                try:
+                    additional_metadata["tokens"]["completion"] = span["attributes"]["llm.token_count.completion"]
+                except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)